main.py

```diff
@@ -9,23 +9,25 @@
 from pyrogram.types import InlineKeyboardMarkup, InlineKeyboardButton, Message
 from bs4 import BeautifulSoup
 import requests
+import os
+from telethon.sync import TelegramClient as Client
```

Comment:
Does telethon Client and Pyrogram Client Do the same job?

Reply:
Yes, but now I see that pyrogram.types documentation is better, I will remove telethon.sync.